modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3902,8 +3902,8 @@ private void addProducesInfo(OpenAPI openAPI, ApiResponse inputResponse, Codegen
 
         int count = 0;
         for (String key : produces) {
-            // escape quotation to avoid code injection, "*/*" is a special case, do nothing
-            String encodedKey = "*/*".equals(key)? key : escapeText(escapeQuotationMark(key));
+            // escape quotation to avoid code injection, "*/*" is a special case, do nothing except adding blank
+            String encodedKey = "*/*".equals(key)? "* / *" : escapeText(escapeQuotationMark(key));
```

Comment:
@Lv-Yi thanks for the fix but I'm afraid we need to keep it as previously reported by other users "*/*" should not be escaped/changed in order for it to function properly in the auto-generated code.

I think we'll need to create additional vendor extensions (e.g. x-consumes-doc) in postProcessOperations to handle "*/*" specifically in  the PHP documentation.

Reply:
@Lv-Yi suggestion: what about doing it in the postProcessOperations of PHP Lumen generator (java class) instead so that other generators won't be impacted by the fix? Here is an example:

https://github.com/OpenAPITools/openapi-generator/pull/354/files#diff-be240257b80dee7947c87e9322ba4950R332